include/seqan3/utility/views/interleave.hpp

```diff
@@ -327,7 +326,7 @@ namespace seqan3::views
  * \header_file{seqan3/utility/views/interleave.hpp}
  *
  * This view can be used to insert one range into another range at regular intervals. It behaves essentially like
- * `| std::views::chunk(step_size) | views::join(inserted_range)` except that for input that models
+ * `| seqan3::views::chunk(step_size) | seqan3::views::join_with(inserted_range)` except that for input that models
```

Comment:
`join_with` doesn't exists. This will be fixed #2526 which will change it to `std::views::join`

Reply:
Hmm the code still uses join_with. Why would std::views::join be more correct?

Reply:
but it will right?

Reply:
I set https://github.com/seqan/seqan3/pull/2526 as  blocking for this PR; so if https://github.com/seqan/seqan3/pull/2526 is merged the comment should be correct, right?

Reply:
I didn't see the blocking PR, sorry. Yes, in that case it would be correct. But I don't know if  `std::views::join` should be preferred. #2526 replaces every call of `join_with` with `std::views::join` if no delimiter is being used.

Reply:
Yeah, but here the semativ is to add a delimiter, right?

Reply:
Yes you are right! That means my change in #2526 is wrong :upside_down_face: